### DISCUSSION TITLE
Support intraday backups?

### DISCUSSION THREAD

**Discussion by ImTheDeveloper** [prodrigestivill/docker-postgres-backup-local#107] [Q&A] [answered]
Will the current script support intraday backups? I was hoping to run every 6 hours giving me 4 per day but wanted to check first.

I can't quite see whether the daily clean up would remove some files but I'm assuming not given there is an HH - mm timestamp on files created on the day

**Comment by prodrigestivill** [comment, marked as answer]
Yes, as states in the README.md file you can define any interval or a set of hours.

It accepts the [cron format](https://en.wikipedia.org/wiki/Cron) well documented anywhere internet or the go cron [specific formats](https://godoc.org/github.com/robfig/cron#hdr-Predefined_schedules).

For every 6 hours you can use: `SCHEDULE=@every 6h`
